src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -116,6 +126,26 @@ public void setUpOrgTestData() {
         }
     }
 
+    public void setUpOrgTestDataForGroupAccess() {
```

Comment:
this method is duplicate, you can use setUpOrgTestData() which is doing the same

Reply:
Thank you for review. Its fixed now. please review.